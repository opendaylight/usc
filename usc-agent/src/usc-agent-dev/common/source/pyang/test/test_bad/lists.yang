module lists {
  namespace "urn:test:lists";
  prefix "lists";
	
  list interface {
    key "name observed-speed";
    config true;

    leaf name {
      type string;
    }

    leaf observed-speed {
      type uint32;
      config false; // error: bad config
    }
  }
  
}
