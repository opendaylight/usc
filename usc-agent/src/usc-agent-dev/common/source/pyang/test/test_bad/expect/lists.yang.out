lists.yang:15: error: KEY_BAD_CONFIG
